.gitignore

```diff
@@ -302,7 +302,7 @@ crashlytics.properties
 crashlytics-build.properties
 
 ### NetBeans ###
-nbproject/private/
+nbproject/
```

Comment:
we should ignore all projects, not only private files, like for `.idea`

Reply:
You must do it in another commit / pull request, it has no aim regarding the title and the description :/

Reply:
PR here - https://github.com/PrestaShop/PrestaShop/pull/21076, this PR also updated to contain only the described change